jaxrs/src/main/java/com/cerner/beadledom/jaxrs/exceptionmappers/ThrowableExceptionMapper.java

```diff
@@ -0,0 +1,51 @@
+package com.cerner.beadledom.jaxrs.exceptionmappers;
+
+import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+
+import com.cerner.beadledom.jaxrs.models.JsonError;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Exception Mapper for all unhandled Exceptions thrown by the service.
+ *
+ * @author Brian van de Boogaard
+ * @author Cal Fisher
+ */
+@Provider
+public class ThrowableExceptionMapper implements ExceptionMapper<Throwable> {
+
+  private static final Logger logger = LoggerFactory.getLogger(ThrowableExceptionMapper.class);
+
+  /**
+   * Generates a Response object with a Status of 500, Content-Type 'application/json', and a
+   * JsonError entity containing details about the unhandled exception in Json format.
+   *
+   * @param exception the Throwable exception that was not handled
+   * @return a json response with status 500
```

Comment:
should we just say something like

```java
@return a {@link JsonError} containing a {@link Response.Status.INTERNAL_SERVER_ERROR}
```

I'm not actually sure if that second `@link` works or not. 😬

Reply:
How does this look? https://github.com/cerner/beadledom/pull/45/commits/865a017a55ce93a2c50c539260b17f6bbcdfe110#diff-8fa995d7fc0ffa3c90069a9fc131c59bR36